README.md

```diff
@@ -40,6 +40,13 @@ curl https://storage.googleapis.com/git-repo-downloads/repo > ~/bin/repo
 chmod a+x ~/bin/repo
 ```
 
+The repo tool depends on python2, so if you installed repo using the curl
```

Comment:
It does not, it works perfectly fine with python 3, but it requires `python` in the environment (it actually tries to switch to python 3 if is started with python 2).

ONL build will likely fail tho if python isn't python2.

Reply:
Well, the fix works. Let me know  iif you want  to update the explanation.